.gitignore

```diff
@@ -15,6 +15,8 @@ package-lock.json
 public
 docs_test
 
+### Antora
+build/
```

Comment:
```suggestion

### Antora
build/
docs_test
```

I already had the `docs_test` folder that I usually used when building documentation locally. I think that it is what I referred to in the documentation, so if using that command there should be no problem already.

That said, I do not oppose you add `build` also for when people do not define an output folder.

Reply:
Sorry, didn't notice that. 
I am curious, how do you specify the output directory ?